### DISCUSSION TITLE
help setting up Mega 2560

### DISCUSSION THREAD

**Discussion by sergiump** [JAndrassy/ArduinoOTA#243] [Q&A] [answered]
Hello, 
Here I am with a new question, this time regarding a Mega 2560. 
I'm trying to make a similar setup like with the esp32 in my previous post #238 but this time with a Mega.

I added the "my boards" folder as per your instructions but I got stuck at burning the bootloader. I checked the link provided but I need a little more details in order to figure it out.
I do have another board (an UNO) that I could use as ISP (if that is needed) but I don't know what to do next, after i connect the UNO and the MEGA pins.

**Comment by JAndrassy** [comment, marked as answer]
https://docs.arduino.cc/built-in-examples/arduino-isp/ArduinoISP/#use-arduino-as-isp
